operationsgateway_api/src/models.py

```diff
@@ -194,32 +199,31 @@ class DateConverterRange(BaseModel):
     from_: Union[int, datetime] = Field(alias="from")
     to: Union[int, datetime]
 
-    @validator("to")
-    def validate_min_max(cls, value, values):  # noqa: B902, N805
-        if value < values["from_"]:
+    @field_validator("to")
+    def validate_min_max(
+        cls,  # noqa: N805
+        value,
+        values,
+    ) -> Union[int, datetime]:
+        if value < values.data["from_"]:
             raise ModelError("to cannot be less than from value")
         else:
             return value
 
 
 class UserSessionModel(BaseModel):
-    id_: Optional[PyObjectId] = Field(alias="_id")
+    id_: Optional[Annotated[ObjectId, PyObjectId]] = Field(None, alias="_id")
     username: str
     name: str
     summary: str
     timestamp: datetime
     auto_saved: bool
-    session: Dict[str, Any]
-
-    class Config:
-        arbitrary_types_allowed = True
+    session: Dict[str, Any] = Field(None)
```

Comment:
Why has `Field` been added here? Can it be removed?

Reply:
didn't effect the pytest so I removed it